test/test_integration.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'integration_test_helpers'
+require 'yaml'
+
+class TestIntegration < Minitest::Test
+
+  IntegrationTestHelpers.find_integration_tests("0.1.0").map do |test_file|
```

Comment:
Depending on our testing strategy, at some point, this should probably fail loudly if no tests can be found, rather than just not running tests.

🤔 Maybe we go ahead and add that before merge and have an env var to opt-out of the behavior?

Reply:
Yeah, I think that makes sense.